object/cluster_compute_resource.go

```diff
@@ -68,3 +68,17 @@ func (c ClusterComputeResource) AddHost(ctx context.Context, spec types.HostConn
 
 	return NewTask(c.c, res.Returnval), nil
 }
+
+func (c ClusterComputeResource) MoveInto(ctx context.Context, host *HostSystem) (*Task, error) {
```

Comment:
Looks good, but maybe this should be `host ...*object.HostSystem` to support multiple hosts?

Reply:
Of course! I was only using this with a single host so I didn't think of it, but you're right. I'll update it to support more than one host.